examples/vision/metric_learning.py

```diff
@@ -22,290 +24,380 @@
 
 """
 ## Setup
+
+This tutorial will use [Tensorflow Similarity](https://github.com/tensorflow/similarity)
+to learn and evaluate the similarity embedding. The tensorflow similarity
+package provides a number of components that:
+
+* Simplify constructing contrastive models.
```

Comment:
Makes training contrastive model simple and fast?

Reply:
Done